editor-components/call_to_action/call_to_action7/call_to_action7.tsx

```diff
@@ -86,54 +57,45 @@ class CallToAction7Page extends BaseCallToAction {
   }
 
   render() {
-    const styling = {
-      backgroundColor: this.getPropValue("cardBackgroundColor"),
-    };
+    const title = this.getPropValue("title");
+    const image = this.getPropValue("image");
+    const button = this.getPropValue("button");
+    const placeholder = this.getPropValue("placeholder", { as_string: true });
+    const disableAnimation = this.getPropValue("disableAnimation");
+
     return (
-      <div className={this.decorateCSS("container")}>
+      <div className={`${this.decorateCSS("container")}`}>
         <div className={this.decorateCSS("max-content")}>
-          <div className={this.decorateCSS("call-to-action7-page")}>
-            <div className={this.decorateCSS("left-child")}>
-              <h1 className={this.decorateCSS("title")}>
-                {this.getPropValue("title")}
-              </h1>
-              <h3 className={this.decorateCSS("description")}>
-                {this.getPropValue("description")}
-              </h3>
-            </div>
-            <div className={this.decorateCSS("card-child")} style={styling}>
-              {this.getPropValue("item-list").map(
-                (item: any, index: number) => (
-                  <div>
-                    <img
-                      className={this.decorateCSS("image")}
-                      src={item.value[5].value}
-                      width={35}
-                      height={35}
-                      alt=""
-                    />
-                    <h1 className={this.decorateCSS("title-2")}>
-                      {item.value[0].value}
-                    </h1>
-                    <h3 className={this.decorateCSS("leftDescription")}>
-                      {item.value[1].value}
-                    </h3>
-                    <ComposerLink path={item.value[3].value}>
-                      <span className={this.decorateCSS("left-button-text")}>
-                        {item.value[2].value}
-                        <img
-                          alt=""
-                          src={item.value[4].value}
-                          width={15}
-                          height={15}
-                        />
-                      </span>
-                    </ComposerLink>
-                  </div>
-                )
-              )}
-            </div>
-          </div>
+          <section className={`${this.decorateCSS("call-to-action7-container")} ${disableAnimation ? this.decorateCSS("no-animation") : ""}`}>
+            {image && (
+              <img className={this.decorateCSS("image")} src={image} alt="" />
+            )}
+            {(title) && (
```

Comment:
this won't work. because title returns <blinkpage> element. it is always non-empty. 
[see here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-check-if-a-string-prop-is-empty)